lib/abbreviato/truncated_sax_document.rb

```diff
@@ -0,0 +1,194 @@
+require 'nokogiri'
+require 'htmlentities'
+
+class TruncatedSaxDocument < Nokogiri::XML::SAX::Document
+  IGNORABLE_TAGS = %w[html head body].freeze
```

Comment:
We end up without the body (just the inner HTML) at the end, so I think this is safe/ok... see `Content#html`